contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -452,4 +452,24 @@ contract CoreIssuanceOrder is
         // Tally fill in orderFills mapping
         state.orderFills[_order.orderHash] = state.orderFills[_order.orderHash].add(_fillQuantity);
     }
+
+    function getPartialAmount(
```

Comment:
Can this get moved into a lib

Reply:
Sure could was looking for a lib that made sense in our set up so just left it as is. But could move into OrderLibrary I suppose.

Reply:
+1